batch/batch/front_end/front_end.py

```diff
@@ -1658,6 +1658,19 @@ async def ui_get_job(request, userdata, batch_id):
     return await render_template('batch', request, userdata, 'job.html', page_context)
 
 
+@routes.get('/batches/{batch_id}/jobs/{job_id}/log/{container}')
+@web_billing_project_users_only()
+@catch_ui_error_in_dev
+async def ui_get_job(request, userdata, batch_id):  # pylint: disable=unused-argument
```

Comment:
Could you change the function name to `ui_get_job_log`? This looks to be the same as the one above it.